features/step_definitions/opportunities_steps.rb

```diff
@@ -15,6 +15,12 @@
   @result_count = page.first(:css, ".search-summary-count").text.to_i
 end
 
+When "I go to $brief URL address" do |brief|
```

Comment:
This new step has been deleted now and previously existing step works, thanks for spotting that!